tool/release/notes/NotesCreate.kt

```diff
@@ -28,7 +28,7 @@ import kotlin.text.Regex.Companion.escapeReplacement
 
 fun main(args: Array<String>) {
     val bazelWorkspaceDir = Paths.get(getEnv("BUILD_WORKSPACE_DIRECTORY"))
-    val githubToken = getEnv("CREATE_NOTES_TOKEN")
+    val githubToken = getEnv("NOTES_CREATE_TOKEN")
```

Comment:
Rename the environment variable to be consistent with how where the tool is located, which is `//library/tool/notes:create`.